third_party/txt/src/txt/paragraph.cc

```diff
@@ -550,7 +552,7 @@ void Paragraph::Layout(double width, bool force) {
         }
       }
 
-      layout.doLayout(text_ptr, text_start, text_count, text_.size(),
+      layout.doLayout(text_ptr, text_start, text_count, text_size,
```

Comment:
The `travis/format.sh` format checker script is requesting that these arguments be combined onto two lines.

See https://travis-ci.org/flutter/engine/builds/406104163#L534

Reply:
Thank you for reviewing. I will run `clang-format`.